code/go/0chain.net/smartcontract/storagesc/challenge.go

```diff
@@ -151,48 +150,11 @@ func (sc *StorageSmartContract) blobberReward(
 		return err
 	}
 
-	move, err = currency.MinusCoin(move, validatorsReward)
-	if err != nil {
-		return err
-	}
-
-	// for a case of a partial verification
-	blobberReward, err := currency.MultFloat64(move, partial) // blobber (partial) reward
-	if err != nil {
-		return err
-	}
-
-	back, err := currency.MinusCoin(move, blobberReward) // return back to write pool
+	blobberReward, err := currency.MinusCoin(move, validatorsReward)
 	if err != nil {
 		return err
 	}
 
-	if back > 0 {
```

Comment:
are we no longer move tokens back?

Reply:
Please check : https://0chain.slack.com/archives/C03788G8N2D/p1697365080962889